### PR TITLE
Improved and completed dutch translation

### PR DESCRIPTION
**Pull request by jghaanstra**
Improved and completed dutch translation

### CODE REVIEW

File: retro-board-app/src/translations/nl.ts [hunk 1]
```diff
@@ -10,167 +10,175 @@ export default {
     header: 'Kies een taal',
   },
   Main: {
-    hint:
-      'Je kan anderen uitnodigen voor deze sessie door de URL te kopiëren en te plakken',
+    hint: 'Je kan anderen uitnodigen voor deze sessie door de URL te kopiëren en te plakken',
   },
   Home: {
-    welcome: undefined,
+    welcome: (userName: string) => `Welkom, ${userName}`,,
```
Comment (antoinejaussoin) [inline retro-board-app/src/translations/nl.ts:16]: You have an extra comma at the end here :)

**Review by antoinejaussoin — CHANGES_REQUESTED** [review]
Thanks so much for your work! One issue with an extra comma (which fails the build), but once that's fixed, I'm merging :) Thanks again

**Comment by jghaanstra** [comment]
Hmm, missed that. It's fixed now.

**Comment by jghaanstra** [comment]
I recreated the containers and even redownloaded release 3.1.1 from Github but my translated strings dont show up. When looking at the code I also see it still says 3.1.0. What am I doing wrong?
```
window.__env__={GOOGLE_ANALYTICS_ID:"UA-17xxxx47-1",SENTRY_URL:"",GIPHY_API_KEY:"",AUTH_GOOGLE_ENABLED:"true",AUTH_TWITTER_ENABLED:"false",AUTH_GITHUB_ENABLED:"false",VERSION:"3.1.0"}
```

**Comment by antoinejaussoin** [comment]
> I recreated the containers and even redownloaded release 3.1.1 from Github and started from scratch but my translated strings dont show up. When looking at the code I also see it still says 3.1.0. What am I doing wrong?
> 
> ```
> window.__env__={GOOGLE_ANALYTICS_ID:"UA-17xxxx47-1",SENTRY_URL:"",GIPHY_API_KEY:"",AUTH_GOOGLE_ENABLED:"true",AUTH_TWITTER_ENABLED:"false",AUTH_GITHUB_ENABLED:"false",VERSION:"3.1.0"}
> ```

I've checked the containers (on prod, www.retrospected.com) and the latest (and tag 3.1.1) seem to be working correctly, they do have your translations:

Retrospected_-_A_fun_way_to_retrospect

Perhaps you didn't pull the latest version correctly? Have you tried `docker-compose pull` ?

**Comment by antoinejaussoin** [comment]
Actually the backend image failed to build, but that shouldn't matter as your translations are on the frontend

**Comment by jghaanstra** [comment]
Yeah, my bad. The NAS running Docker was having DNS issues which did not result in any errors but also didnt pull in the latest image. It's fixed now, thanx.

**Pull request merged**